.packit.yaml

```diff
@@ -0,0 +1,54 @@
+actions:
+  create-archive:
+    - './autogen.sh --enable-modules'
+    - 'make'
+    - 'make dist'
+    - 'bash -c "ls *.tar*"'
+
+jobs:
+- job: copr_build
+  owner: "@storage"
+  project: udisks-pr
```

Comment:
@vojtechtrefny Just one question: Why do you want a custom COPR for PR builds? Aren't the transient ones that packit provides good enough for testing PR? (We only use them in cockpit, and they are just fine).

Note that this is not the same as your "daily" COPR -- that should be a permanent one, but that will have `trigger: push` (on master).

Reply:
Oh, I forgot about this. The packit team recommended using a custom repo and adding the `@storage/udisks-daily` to the *external repositories*. This also didn't work and since that I removed the repo from `@storage`. (This also explains why Packit failed to do a new build when I triggered it manually.)